test/plugins/base.plugin.bats

```diff
@@ -1,11 +1,14 @@
-#!/usr/bin/env bats
+# shellcheck shell=bats
 
-load ../test_helper
-load ../test_helper_libs
-load ../../plugins/available/base.plugin
+load "${MAIN_BASH_IT_DIR?}/test/test_helper.bash"
+
+function local_setup_file() {
+  setup_libs "helpers"
+  load "${BASH_IT?}/plugins/available/base.plugin.bash"
+}
 
 @test 'plugins base: ips()' {
-  if [[ $CI ]]; then
+  if [[ -n "${CI:-}" ]]; then
     skip 'ifconfig probably requires sudo on TravisCI'
```

Comment:
we might just remove this part, we are not on travis anymore and  I am not sure this is the same in github actions

Reply:
GitHub documentation says that `sudo` *is* available and won't prompt for password, so yeah. It can go!